prow/oss/plugins.yaml

```diff
@@ -346,6 +347,15 @@ plugins:
     - verify-owners   # Validates OWNERS file changes in PRs.
     - wip   # Applies a label to PRs with wip in the title to block merge
 
+repo_milestone:
+  kubeflow/trainer:
+    maintainers_team: kubeflow-trainer-maintainers
+    maintainers_friendly_name: Kubeflow Trainer Maintainers
+
+milestone_applier:
+  kubeflow/trainer:
+    master: v2.0
```

Comment:
It looks like you added this in the middle of the `plugins` enablement config. Please instead add this above the plugin enablement (above line 86).